src/Redux/details/details.js

```diff
@@ -0,0 +1,71 @@
+import axios from 'axios';
+
+const FETCH_DETAILS_REQUEST = 'detailsStore/details/fetch_request';
+const FETCH_DETAILS_SUCCESS = 'detailsStore/details/fetch_success';
+const FETCH_DETAILS_FAILURE = 'detailsStore/details/fetch_failure';
+const FORECAST_URL = 'https://api.open-meteo.com/v1/forecast?';
+const initialiState = {
+  loading: false,
+  error: '',
+  data: [],
+};
+
+export const fetchForecastRequest = () => ({
+  type: FETCH_DETAILS_REQUEST,
+});
+
+export const fetchForecastSuccess = (payload) => ({
+  type: FETCH_DETAILS_SUCCESS,
+  payload,
+});
+
+export const fetchForecastFailure = () => ({
+  type: FETCH_DETAILS_FAILURE,
+});
+
+export const fetchCityForecast = (lat, lng, cityName) => (dispatch) => {
+  dispatch(fetchForecastRequest());
+  axios.get(`${FORECAST_URL}latitude=${lat}&longitude=${lng}&hourly=temperature_2m`, { headers: {} })
+    .then((response) => {
+      const { data } = response;
+      const { hourly } = data;
+      /* eslint-disable-next-line */
```

Comment:
- [ ] Please remove this eslint diable line its best to fix the issue in the code.